test/integration/connect/test_connect.py

```diff
@@ -138,6 +139,37 @@ def test_connect_to_localhost_app(tt_cmd, tmpdir):
     stop_app(tt_cmd, tmpdir, "test_app")
 
 
+def test_connect_to_localhost_app_credentials(tt_cmd, tmpdir):
+    empty_file = "empty.lua"
+    # The test application file.
+    test_app_path = os.path.join(os.path.dirname(__file__), "test_localhost_app", "test_app.lua")
+    # The test file.
+    empty_file_path = os.path.join(os.path.dirname(__file__), "test_file", empty_file)
+    # Copy test data into temporary directory.
+    copy_data(tmpdir, [test_app_path, empty_file_path])
+
+    # Start an instance.
+    start_app(tt_cmd, tmpdir, "test_app")
+
+    # Check for start.
+    file = wait_file(tmpdir, 'ready', [])
```

Comment:
Is your branch rebased with master? 
If no, this test may fail.
Solution: replace tmpdir with tmpdir_with_cfg

Reply:
Thank you, rebased.